plugins/media.js

```diff
@@ -20,6 +20,12 @@ export function isHTMLAudio(media) {
   return ['audio/flac', 'audio/ogg', 'audio/mpeg'].includes(media.ebucoreHasMimeType);
 }
 
+export function isPlayableMedia(media) {
+  return (media.ebucoreHasMimeType.startsWith('video/') ||
```

Comment:
This needs also to detect and return true for EUscreen videos. Check the URL against `http://www.euscreen.eu/item.html*`